![alt text](https://courses.edx.org/assets/courseware/v1/92f8defb29778209ffbeccc4d996d27e/asset-v1:UTAustinX+UT.7.11x+2T2017+type@asset+block/cycling.jpg)


**In 2011, researchers at the Texas Transportation Institute and the Center for Transportation Research at UT Austin ran an advertising campaign aimed at recruiting Austin cyclists to join the South Congress Bike Mapping Project.  As members of the project, cyclists downloaded and used Cycle Tracks, a smartphone app developed by the San Francisco County Transportation Authority to track where people are riding their bikes based on their GPS points.  The goal was to gain new information about bike commuting patterns and this data set is based on the results of the study: 3600 trips tracked from 315 users over a 6 month period.   Data includes distances traveled, speed of travel, and reasons for travel among other variables.**

>Primary Research Question 

#### How many of the cyclists rode daily, how many of these riders were male and how many female, and what were their average ages?

In [1]:
#Input data
data <- read.csv("BikeData.csv")


### Make a table of cyclists frequency rides


In [2]:
 table(data$cyc_freq)


                  Daily  Less than once a month Several times per month 
                     47                       2                      14 
 Several times per week 
                     58 

 So there are **47** daily riders.

### Make a subset for male and female riders

In [3]:
males <- data[data$gender == 'M',]
females <- data[data$gender == 'F',]

### Table of cyclist frequency rides for males and females 

In [4]:
table(males$cyc_freq)




                  Daily  Less than once a month Several times per month 
                     38                       0                       9 
 Several times per week 
                     43 

In [5]:
table(females$cyc_freq)


                  Daily  Less than once a month Several times per month 
                      9                       2                       5 
 Several times per week 
                     15 

So number of males daily riders is **38** and number of females daily riders is **9**

### Calculate Age

In [6]:
#Calculate average age of all daily riders 
mean(data$age[data$cyc_freq=="Daily"])

[1] 33.65957

So average age of daily riders is **33.7**

In [7]:
#Calculate average age of males daily riders 
mean(males$age[males$cyc_freq=="Daily"])

[1] 33.92105

So average age of males daily riders is **33.9**

In [8]:
#Calculate average age of females daily riders 
mean(females$age[females$cyc_freq=="Daily"])

[1] 32.55556

So average age of males daily riders is **32.6**

### Conclusion

In this lab, we examined data on **47** daily riders. Most of the daily riders **38** were male.The average age of all daily riders was 
**33.7**.The average age of female daily riders was **32.6**  , while the averge age of male daily riders was **33.9**
 